ansible/roles/promtail/tasks/main.yml

```diff
@@ -0,0 +1,96 @@
+- name: Install dependencies
```

Comment:
Gripe: this is just a lisp in yaml cloths. How did we go so wrong....

Reply:
It gets worse. Ansible blends Bash _and_ Python to it too. 😭